framework/db/Connection.php

```diff
@@ -618,9 +624,10 @@ protected function initConnection()
      */
     public function createCommand($sql = null, $params = [])
     {
-        $command = new Command([
+        $command = Yii::createObject([
```

Comment:
What is the benefit of calling `Yii::createObject` instead of `new $this->commandClass(...` ? It adds a lot of overhead that is not really needed in this case imo.